src/api/github/checkRun.ts

```diff
@@ -105,7 +107,7 @@ export const createCheckRun = async ({
   ]);
 
   const client = new Octokit({
-    auth: process.env.GITHUB_API_TOKEN ?? process.env.GITHUB_TOKEN,
```

Comment:
Thought this was getting a little tedious, as I had to do the same in my Changesets PR.